src/components/pages/room.js

```diff
@@ -67,13 +68,15 @@ export default function Room({ PageTemplate, initialState }) {
     confirmRoomDelete(t, room.name, handleRoomDelete);
   };
 
-  const handleInvitationModalClose = wasNewInvitationCreated => {
+  const handleInvitationModalClose = newInvitation => {
     setIsRoomInvitationModalVisible(false);
-
-    if (!wasNewInvitationCreated) {
-      return;
+    if (newInvitation) {
+      setInvitations(currentInvitations => {
+        const invitationsByEmail = new Map(currentInvitations.map(x => [x.email, x]));
+        invitationsByEmail.set(newInvitation.email, newInvitation);
+        return [...invitationsByEmail.values()].sort(by(x => x.sentOn));
```

Comment:
Nice touch, showing just the lates invitation

Reply:
Actually, I show all, but there can never be two with the same email (the server deletes the older one), so that's what I'm doing here also in the frontend.